pkg/cluster/controllers/database/readiness.go

```diff
@@ -106,22 +102,27 @@ func getDatabasePasswordRefName(name string) string {
 }
 
 // DeployComponentSecret deploy harbor component database secret.
-func (p *PostgreSQLController) DeployComponentSecret(ctx context.Context, conn *Connect, ns string, secretName string) (*corev1.Secret, error) {
+func (p *PostgreSQLController) DeployComponentSecret(ctx context.Context, conn *Connect, ns string, secretName string,
```

Comment:
If the whole `*goharborv1.HarborCluster` has been passed into the func arguments, the namespace arg can be removed, right?  @Ghostbaby

Reply:
yes.